pkg/openapi/crdSync.go

```diff
@@ -54,7 +84,7 @@ func (c *crdSync) Run(workers int, stopCh <-chan struct{}) {
 	c.sync()
 
 	for i := 0; i < workers; i++ {
-		go wait.Until(c.sync, time.Second*25, stopCh)
+		go wait.Until(c.sync, 10*time.Minute, stopCh)
```

Comment:
Can we pull all constants into a common file? Ideally, we can make these configurable over time.